devtools/conda-envs/openff.yaml

```diff
@@ -1,8 +1,7 @@
 name: qcarchive
 channels:
   - defaults
-  - omnia
-  - psi4/label/dev
```

Comment:
I think we should leave in psi4/label/dev and test against the current master version. Right @loriab ?
I suppose we can leave the pin below